src/monattack.cpp

```diff
@@ -165,7 +182,7 @@ void mattack::acid(monster *z, int index)
 
     int t;
     int junk = 0;
-    if( !z->sees( t, target ) ||
+    if( !z->sees( target, t ) ||
```

Comment:
I wrote the monster::sees basing it on sees_player just below. sees_player had an optional argument, which had to be last argument and so had argument order different from all other "sees" function. monster::sees has no optional argument, so I changed it to look like all other "sees" function.